src/frame/reason.rs

```diff
@@ -22,8 +22,12 @@ pub enum Reason {
 
 // ===== impl Reason =====
 
+const CONNECT_ERROR_MSG: &'static str = {
+    "connection established in response to a CONNECT request was reset or abnormally closed"
+};
```

Comment:
A block containing just the static string? Seems odd...

Reply:
yep, fixed